content/articles/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/index.md

```diff
@@ -0,0 +1,79 @@
+
+### Overview
+This article will cover:
+- [How the private FM broadcast will work](#how-the-private-fm-broadcast-will-work)
+- [Setting up the Raspberry](#setting-up-the-raspberry)
+- [Conversion of PRi to Private FM transmitter](#conversion-of-pri-to-private-fm-transmitter)
+- [Conclusion](#conclusion)
+- [Relevant resources](#relevant-resources)  
+  
+### How the private FM broadcast will work
+To reduce electromagnetic interference, all microprocessors are fitted with a synchronous digital system. Using a spread-spectrum clock signal or what is referred to as SSCS in short form, suppression of electromagnetic interference is achieved. This signal has a frequency of between 1MHz and 250MHz which falls between the FM band. Therefore, by using a program that modulates frequency, the Pi will be tweaked to operate as a radio transmitter. This signal will then be produced via the GPIO pin four of the Pi. A short wire of around twenty centimeters can be attached to act as an antenna.
+
+### Setting up the Raspberry
+Since the Raspberry Pi has a running operating system, boot it and connect the output HDMI to a monitor. Also connect a keyboard and mouse. On accessing the Desktop, search for the network option which will enable connection to the router (internet). Next, move on to the pi configuration through the menu and enable SSH communication. 
+Once this is enabled, we can now access the Raspberry using Putty as long the personal computer and the Pi are connected to the same network. Open Putty on the personal computer and key in the IP address of the Raspberry Pi to connect via SSH. A session will then appear and ask for a login credentials. The credentials to be used here are pi for username and raspberry for password. 
+
+### Conversion of PRi to Private FM transmitter
+*Step One:*
+
+In the current directory, create a new folder that we will use to store all the files required to get the radio broadcast working. In this case, we will create a folder called Private_Radio and then navigate inside that folder using the following commands:
+```JSON
+mkdir Private_Radio
+cd Private_Radio
+```
+*Step Two:*
+
+This stage involves cloning the code from GitHub. In order to achieve this, we use the git clone commands as shown below.
+```JSON
+sudo git clone https://github.com/markondej/fm_transmitter
+```
+*Step Three:*
+
+This code is written in C. This, therefore, calls for the use of compilers to compile it. In this guide, we will use a program known as gcc and g++. The compiler to be used is known as *make*. The following commands will be used to download the compilers.
```

Comment:
In this section, we are compiling the code written in C to which language? Please make it clear for the reader